src/index.js

```diff
@@ -32,16 +32,16 @@ _adapters._date.override({
 		return FORMATS;
 	},
 
-	parse: function(value/* , format */) {
+	parse: function(value) {
 		if (helpers.isNullOrUndef(value)) {
 			return null;
 		}
-
-		return parse(value);
+		value = parse(value);
+		return isValid(value) ? value.valueOf() : null;
 	},
 
 	format: function(time, fmt) {
-		return format(time, fmt);
+		return format(time, fmt, this.options || {});
```

Comment:
`|| {}` is useless because [it's already done in the constructor](https://github.com/chartjs/Chart.js/blob/master/src/core/core.adapters.js#L36).